Documentation/Summary/Index.rst

```diff
@@ -133,26 +133,20 @@ Tutorial - Sitepackages - Part 1 of 3
 
 `YouTube: Part 1 of 3 <https://www.youtube.com/watch?v=HtBmim7pc0o>`__
 
-.. only:: html, singlehtml
-
-   .. youtube:: HtBmim7pc0o
+.. youtube:: HtBmim7pc0o
 
 
 Tutorial - Sitepackages - Part 2 of 3
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 `YouTube: Part 2 of 3 <https://www.youtube.com/watch?v=deSMVfCSCXY>`__
 
-.. only:: html, singlehtml
```

Comment:
why are you removing the html/singlehtml directive? in the print versions the youtube videos don't make sense.

Reply:
reverted it